sphinx_rtd_theme/layout.html

```diff
@@ -26,7 +26,11 @@
     {%- if css|attr("rel") %}
       <link rel="{{ css.rel }}" href="{{ pathto(css.filename, 1) }}" type="text/css"{% if css.title is not none %} title="{{ css.title }}"{% endif %} />
     {%- else %}
```

Comment:
The logic here is a bit messed up I'm starting to realize it is a bit more complicated than I originally thought.

```
 <link rel="stylesheet" href="{{ pathto('_static/' + style, 1) }}" type="text/css" />
  <link rel="stylesheet" href="{{ pathto('_static/pygments.css', 1) }}" type="text/css" />
  {%- for css in css_files %}
    {%- if sphinx_version >= "1.8.0" %}
      {%- if css|attr("filename") %}
        {{ css_tag(css) }}
      {%- else %}
         <link rel="stylesheet" href="{{ pathto(css, 1) }}" type="text/css" />
      {%- endif %}
    {%- else %}
      {%- if css|attr("rel") %}
        <link rel="{{ css.rel }}" href="{{ pathto(css.filename, 1) }}" type="text/css"{% if css.title is not none %} title="{{ css.title }}"{% endif %} />
      {%- else %}
        <link rel="stylesheet" href="{{ pathto(css, 1) }}" type="text/css" />
      {%- endif %}
    {%- endif %}
  {%- endfor %}
```

Reply:
I see `filename` attribute already present on 1.8.0, so why is the additional check needed?

Reply:
Not sure but this is what sphinx does: https://github.com/sphinx-doc/sphinx/blob/3.x/sphinx/themes/basic/layout.html#L97

The case for before 1.8 is to match the template before this commit: https://github.com/sphinx-doc/sphinx/commit/c5d6942b88a5902a843ba41c74cb01c402413e48

Reply:
ok, should be ready now

Reply:
@Blendify ping